src/client/client4.ts

```diff
@@ -2276,6 +2279,20 @@ export default class Client4 {
         );
     }
 
+    setFirstAdminVisitMarketplaceStatus = async () => {
+        return this.doFetch<StatusOK>(
+            `${this.getBaseRoute()}/first_admin_visit_marketplace`,
+            {method: 'post', body: JSON.stringify({first_admin_visit_marketplace_status: true})},
```

Comment:
Strictly speaking the server doesn't care about the payload. Should we not send one then?

Reply:
While more verbose, I find it easier to follow - maps to how we store the state of the flag in the store (as opposed to interpreting the call at the server as "true" and doing the conversion at the receiving end). Also, it keeps the door open to more interesting scenarios in the future.

Reply:
The server does ignore the payload. Hence, the any call to the server is currently interpreted as "true", correct?

Also: How does ignoring the payload closes a door to future feature? It seems the other way around: If we are now conservative with sending the payload, we can add semantics to the payload later.

Reply:
"Hence, the any call to the server is currently interpreted as "true", correct?" - not disputing that. As I said, I find it easier to follow - also, to me it aligns better with the next layer - don't have to convert the "true" into a  name-value pair.
My second comment was more about - if we ever send the "false"/other state, the door is already opened (still need to do a few changes).